src/integration/java/io/pinecone/integration/controlPlane/ConfigureIndexTest.java

```diff
@@ -163,13 +133,17 @@ public void sizeIncrease() {
             // Get the index description to verify the new pod type
             indexMeta = indexOperationClient.describeIndex(indexName);
             assertEquals("p1.x2", indexMeta.getDatabase().getPodType());
+
+            // Delete this index since it'll be unused for future tests
+            indexOperationClient.deleteIndex(indexName);
```

Comment:
Index of pod-type p1.x2 i.e. scaled up is useless for future tests, so delete them right away.